config/locales/rails_admin.en.yml

```diff
@@ -94,6 +94,7 @@ en:
         menu: "Delete"
         breadcrumb: "Delete"
         link: "Delete '%{object_label}'"
+        link_with_number: "Delete '%{object_label}' #%{object_number}"
```

Comment:
Since this is not to be used in the Delete action, how do you think of creating this under `admin.form`?

```yaml
admin:
  form:
    delete_file: "Delete '%{field_label}' #%{number}"
```